oncotator/oncotator-index.py

```diff
@@ -97,7 +99,7 @@
 
             tsv_data = parse_cosmic_line(line)
             try:
-                chromosome, coords = tsv_data['Mutation GRCh37 genome position'].split(':')
+                chromosome, coords = tsv_data['Mutation genome position'].split(':')
```

Comment:
I'm assuming this is version dependent?  Should there be an upfront check for valid versions?  Should this an uncaught exception with a warning message in case someone uses an old cosmic?